### DISCUSSION TITLE
Education application is approved but not get benefit

### DISCUSSION THREAD

**Discussion by Manpreet0980** [community/community#138108] [GitHub Education]
### Hi everyone,

 I am approved my application and few days ago and my benefit not available and plz sort out quickly

**Comment by davevad93** [comment]
Hi @Manpreet0980 ,

GitHub no longer provides individual manual reviews and status updates.

You can check the status of your application [here](https://education.github.com/discount_requests/pack_application), when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Manpreet0980 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
